README.md

```diff
@@ -1,10 +1,15 @@
 # Coder Docs
 
-This repository contains the documentation source files for [coder.com/docs](https://coder.com/docs).
+This repository contains the documentation source files for
+[coder.com/docs](https://coder.com/docs).
 
-Pre-production documentation is drafted in a private repository before being
-synchronized with this repository as releases are made public.
+Is something confusing, unclear, or just want to contribute something new?
+[Open an issue or pull request](https://github.com/cdr/docs/issues/new)!
 
-Interested in using Coder? [Contact our Technical Sales Team](https://coder.com/contact).
+Contributions can be previewed in the context of the site using pull request
+deploy previews. :rocket:
 
-See something wrong or unclear? Please contribute by [opening an issue or pull request](https://github.com/cdr/docs/issues/new).
+Our documentation is published once a day. Expect any merged contributions to
+be live within 24 hours.
+
+**Interested in using Coder?** [Contact our Technical Sales Team](https://coder.com/contact).
```

Comment:
Thoughts on the following?

# Coder Docs

This repository contains the documentation source files for
[coder.com/docs](https://coder.com/docs).

## Contributing

Is something confusing, unclear, or just want to contribute something new?
[Open an issue or pull request](https://github.com/cdr/docs/issues/new)!

Contributions can be previewed in the context of the site using pull request
deploy previews. :rocket:

Our documentation is published once a day. Expect any merged contributions to
be live within 24 hours.

## Interested in using Coder?

[Contact our Technical Sales Team](https://coder.com/contact).

Reply:
oh yeah, that's good, i was just working with the original format trying to make contribution situation slightly more clear :) will add that.